src/tox/config/loader/str_convert.py

```diff
@@ -63,6 +63,11 @@ def to_command(value: str) -> Command:
                 pos = splitter.instream.tell()
         except ValueError:
             args.append(value[pos:])
+        if len(args) == 0:
```

Comment:
How is this possible? Do you have a tox.ini repro?

Reply:
Roughly speaking, it happens when a command configuration key (like `install_command`) is non-empty, but none of the entries (lines?) within it match the current environment name.

(for example: the minimal repro example in the description of #2695: `install_command` is configured for environment name `foo`, and then the failure occurs when `tox` runs using the default environment name `py`)